packaging/os/dnf.py

```diff
@@ -19,18 +20,9 @@
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-
-import traceback
 import os
 import dnf
```

Comment:
The previous version was catching the import exception around non-existent code, that had never been part of the DNF API. There's no direct checking whether the `DNF` package can be actually imported, but it sounds like nice functionality. I'll add it. Thanks for the suggestion